[CI] ReadTheDocs PR builds fail

## Expected behavior ##

RTD builds docs on each PR push.

## Actual behavior ##

Currently failing with output

```
asdf install python mambaforge-4.10.3-10
Downloading python-build...
Cloning into '/home/docs/.asdf/plugins/python/pyenv'...
python-build mambaforge-4.10.3-10 /home/docs/.asdf/installs/python/mambaforge-4.10.3-10
Downloading Mambaforge-4.10.3-10-Linux-x86_64.sh...
-> https://github.com/conda-forge/miniforge/releases/download/4.10.3-10/Mambaforge-4.10.3-10-Linux-x86_64.sh
Installing Mambaforge-4.10.3-10-Linux-x86_64...
EnvironmentLocationNotFound: Not a conda environment: /home/docs/checkouts/readthedocs.org/user_builds/mdanalysis/conda/5376
BUILD FAILED (Ubuntu 26.04 using python-build 2.7.3-3-g3974647f)
Inspect or clean up the working tree at /tmp/python-build.20260626180613.118
Results logged to /tmp/python-build.20260626180613.118.log
Last 10 log lines:
Linking cryptography-36.0.0-py39h95dcef6_0
Linking brotlipy-0.7.0-py39h3811e60_1003
Linking conda-package-handling-1.7.3-py39h3811e60_1
Linking pyopenssl-21.0.0-pyhd8ed1ab_0
Linking urllib3-1.26.7-pyhd8ed1ab_0
Linking requests-2.26.0-pyhd8ed1ab_1
Linking conda-4.10.3-py39hf3d152e_4
Linking mamba-0.19.0-py39hfa8f2c8_0
Transaction finished
installation finished.
```



## Code to reproduce the behavior ##

Eg https://app.readthedocs.org/projects/mdanalysis/builds/33328369/

--- a/.readthedocs.yml
+++ b/.readthedocs.yml
@@ -9,16 +9,11 @@ version: 2
 sphinx:
   configuration: package/doc/sphinx/source/conf.py
 
-# Build documentation with MkDocs
-#mkdocs:
-#  configuration: mkdocs.yml
-
 # Set the version of Python and other tools you might need
 build:
   os: ubuntu-lts-latest
   tools:
-    # python: "miniconda3-4.7"
-    python: "mambaforge-4.10"
+    python: "miniforge3-latest"
 
 # Optionally set the version of Python and requirements required to build your docs
 python:

